demo.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Demo Grapher</title>
+    </head>
+    <body>
+        <p>
+            Enter a JS math equation<br>using x, y, k, and theta:
+        </p>
+        <div style="color: white; z-index: 5;">
+            <input style="border-radius: 1rem;" id="ieq" value="Math.sin(k*x)*Math.cos(y/k)" type="text">
+            <br>
+            <button style="border-radius: 1rem;" onclick="handleInput();" type="button" id="1">Save</button>
+            <button style="border-radius: 1rem; display:none;" onclick="cancel(); handleInput();" type="button" id="2">Save</button>
+        </div>
+        <canvas style="width: 80%; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);" id="bg">Start streaming on a modern browser!</canvas>
+    </body>
+    <script>
+        var equation
```

Comment:
Wouldn't it be better to do something like:
```suggestion
        let equation
```

Reply:
Done